health/health.go

```diff
@@ -36,25 +36,21 @@ type Server struct {
 	mu sync.Mutex
 	// statusMap stores the serving status of the services this Server monitors.
 	statusMap map[string]healthpb.HealthCheckResponse_ServingStatus
+	updates   map[string]map[healthpb.Health_WatchServer]chan healthpb.HealthCheckResponse_ServingStatus
 }
 
 // NewServer returns a new Server.
 func NewServer() *Server {
 	return &Server{
-		statusMap: make(map[string]healthpb.HealthCheckResponse_ServingStatus),
+		statusMap: map[string]healthpb.HealthCheckResponse_ServingStatus{"": healthpb.HealthCheckResponse_SERVING},
+		updates:   make(map[string]map[healthpb.Health_WatchServer]chan healthpb.HealthCheckResponse_ServingStatus),
 	}
 }
 
 // Check implements `service Health`.
 func (s *Server) Check(ctx context.Context, in *healthpb.HealthCheckRequest) (*healthpb.HealthCheckResponse, error) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
-	if in.Service == "" {
```

Comment:
Why is this deleted?

Reply:
@dfawley realized this. We were always returning SERVING for "". This shouldn't be the case.